doc/user/content/get-started/quickstart.md

```diff
@@ -303,8 +303,7 @@ DROP SOURCE auction_house CASCADE;
 DROP TABLE fraud_accounts;
 ```
 
-## What's next?
+
```

Comment:
Yup -- I only deleted this since at this point in time since there wasn't any content once I removed the convert playground to trial account CTA.

I can definitely add the two CTAs to keep the section.

Reply:
For context, we previously did not allow playground accounts to create connections and sources; these accounts were limited to load generators. We can now point users to getting started with their own data, since the restriction no longer applies for trial accounts.

Reply:
Ah ... got it. For now, just added 2 bullet points to click the various buttons in the Console (since one button goes to the Sources in the console - and the other does the chilipiper for the guided tour).  But, definitely will need to be revisited when we do the refresh of the quickstart.